src/dotnet/commands/dotnet-restore/RestoreCommandParser.cs

```diff
@@ -94,8 +94,13 @@ private static Option[] ImplicitRestoreOptions(bool showHelp = false, bool useSh
                     useShortOptions ? "-f|--force" : "--force",
                     LocalizableStrings.CmdForceRestoreOptionDescription,
                     Accept.NoArguments()
-                          .ForwardAs("-property:RestoreForce=true"))
+                          .ForwardAs("-property:RestoreForce=true")),
+                 Create.Option(
```

Comment:
Does the fact that this is in an ImplicitRestoreOptions method mean that this option applies to operations such as `dotnet build` or `dotnet publish`?  I think we had said that it would not.

Reply:
@dsplaisted 
good point...double checked the meeting notes, that was the agreement. 

I'm not sure how make a command not propagate though...can you point me to the code?

Reply:
I'm not familiar myself with how the options/parsing code works.  @wli3 may be able to help.

Reply:
If this is meant only for the `restore` command, then it probably belongs in the Concat call [here](https://github.com/nkolev92/cli/blob/c834c8cd0b147e5232460174a1d6e89310b7a9e1/src/dotnet/commands/dotnet-restore/RestoreCommandParser.cs#L27).

Reply:
Thanks @peterhuene, that worked. 

It's a bit difficult to understand how the differentiation is being made between those 2 before I saw the CreateWithRestoreOptions helper.

Reply:
Yeah, this could use a little refactoring for comprehensibility's sake.